docs/README.md

```diff
@@ -56,4 +56,3 @@
   * [spaceship::displaytime](/docs/API.md#spaceshipdisplaytime-seconds)
   * [spaceship::union](/docs/API.md#spaceshipunion-arr1-arr2-)
 * [Troubleshooting](/docs/Troubleshooting.md)
-
```

Comment:
Do not remove the last empty line. You can use [`.editorconfig`](http://editorconfig.org/) which is already present in our repo.

Reply:
Reverted to original file.